src/SfdxProjectBuilder.groovy

```diff
@@ -763,6 +764,24 @@ class SfdxProjectBuilder implements Serializable {
     }
   }
 
+  private void isPassingCodeScan()
+  {
+    
+    _.echo("running sfdx scanner with default settings, fail on severity >= 2")
+
+    def rmsg
+    def jsonParsedResponse
+      rmsg =_.sh returnStdout: true, script: "sfdx scanner:run --target sfdx-source/ --severity-threshold 2"
+      _.echo( rmsg )
+      jsonParsedResponse = jsonParse(rmsg)
+      _.echo("jsonParsedResponse.exitCode (A) == " + jsonParsedResponse.exitCode)
+      _.echo("jsonParsedResponse.name (A) == " + jsonParsedResponse.name)
+      if (jsonParsedResponse.exitCode != 0){
+        _.error("failed with exit code"  + jsonParsedResponse.exitCode)
+      }
+
```

Comment:
I took a look at the json response from the command. I am not seeing "exitCode" nor "name".  Am I missing something?